File: src/main/java/com/todobackend/dropwizard_kotlin/theApplication.java
package com.todobackend.dropwizard_kotlin;

import io.dropwizard.Application;
import io.dropwizard.setup.Bootstrap;
import io.dropwizard.setup.Environment;

public class theApplication extends Application<theConfiguration> {

    public static void main(final String[] args) throws Exception {
        new theApplication().run(args);
    }

    @Override
    public String getName() {
        return "todobackend";
    }

    @Override
    public void initialize(final Bootstrap<theConfiguration> bootstrap) {
        // TODO: application initialization
    }

    @Override
    public void run(final theConfiguration configuration,
                    final Environment environment) {
        // TODO: implement application
    }

}

File: src/main/java/com/todobackend/dropwizard_kotlin/theApplication.kt
package com.todobackend.dropwizard_kotlin

import com.todobackend.dropwizard_kotlin.resources.TodoResource
import io.dropwizard.Application
import io.dropwizard.setup.Bootstrap
import io.dropwizard.setup.Environment

class theApplication : Application<theConfiguration>() {

    override fun getName(): String = "todobackend"

    override fun initialize(bootstrap: Bootstrap<theConfiguration>?) {
        // TODO: application initialization
    }

    override fun run(configuration: theConfiguration,
                     environment: Environment) {
        val todoResource = TodoResource()
        environment.jersey().register(todoResource)
    }
}